### PR TITLE
feat(docs): Box primitive

### PR DESCRIPTION
**Pull request by viktoria-schwarz**
#1110

### CODE REVIEW

File: docs/content/components/box.mdx [hunk 1]
```diff
@@ -2,7 +2,49 @@
 title: Box
 ---
 
-# Box-Component
+# Box
+
```
Comment (ti10le) [inline docs/content/components/box.mdx:6]: I would add here Description headline like in the other components and then link to your box primitive description

File: docs/content/foundation/box-primitive.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    </Box>
+  );
+};
+```
+
+## Custom CSS
+
+Staying inside the boundaries of the design system and its contraints should be the norm. But as limiting styling only
+to allowed values defined by the system can be very restrictive, you can use the `className` prop and generate custom
+styles with the <Link to="../../guides/usestyles">`useStyles` hook</Link>.
```
Comment (ti10le) [inline docs/content/foundation/box-primitive.mdx:62]: Just a little thing. I would take the word hook out of the link. Looks a little bit komisch 😀

File: docs/src/components/Layout.tsx [hunk 3]
```diff
@@ -19,7 +20,12 @@ export const Layout: React.FC = ({ children }) => {
         <title>Marigold Design System</title>
       </Helmet>
       <Columns space={8}>
-        <Column width={2}>
+        <Column
```
Comment (ti10le) [inline docs/src/components/Layout.tsx:23]: oook do it 😂

File: docs/src/components/Navigation/Navigation.tsx [hunk 4]
```diff
@@ -49,7 +49,7 @@ const NavigationSection = ({ name, children }: NavigationSectionProps) => {
 
 export const Navigation: React.FC = () => {
   const tree = useNavigation();
-  console.log(tree);
+  // console.log(tree);
```
Comment (ti10le) [inline docs/src/components/Navigation/Navigation.tsx:52]: Do we need this? I think let it active or remove it. What do you mean?

**Review by ti10le — DISMISSED** [review]

File: docs/src/components/Logo/Logo.tsx [hunk 5]
```diff
@@ -2,5 +2,5 @@ import React from 'react';
 import { StaticImage } from 'gatsby-plugin-image';
 
 export const Logo: React.FC = () => (
-  <StaticImage src="./logo.png" alt="Marigold Logo" />
+  <StaticImage src="./logo.png" alt="Marigold Logo" width={180} />
```
Comment (sebald) [inline docs/src/components/Logo/Logo.tsx:5]: why? :-o

**Review by ti10le — APPROVED** [review]

**Pull request merged**